Automattic/wp-calypso
# Jetpack: update product icons

#### Changes proposed in this Pull Request

* Updates all SVG images to a green circle + product icon.
* Updates product references and adds a couple of new ones: Boost and Social.
* Updates Devdocs documentation and example page.

More context in pbNhbs-2GY-p2
Analog Jetpack PR: https://github.com/Automattic/jetpack/pull/24559


#### Testing instructions

* Fire up this PR.
* Add products to your Jetpack site and ensure the correct icon is visible on http://calypso.localhost:3000/plans/my-plan/SITE_URL
* Additionally, check Devdocs: http://calypso.localhost:3000/devdocs/design/product-icon

#### Screenshots

Before | After
--|--
<img width="1066" alt="image" src="https://user-images.githubusercontent.com/390760/170975179-10937077-bc42-44e1-b890-4c7ea813f93d.png"> | <img width="1064" alt="image" src="https://user-images.githubusercontent.com/390760/170975203-95a8e4b3-933d-4f54-bea1-ca50f7b8be99.png">
<img width="1063" alt="image" src="https://user-images.githubusercontent.com/390760/170975508-04c477e0-491e-4857-aead-31a0917cf883.png"> | <img width="1063" alt="image" src="https://user-images.githubusercontent.com/390760/170975476-94f510a9-d89b-44be-825b-47d7f0304b44.png">
<img width="722" alt="image" src="https://user-images.githubusercontent.com/390760/170974717-38cc57ef-fb6e-4379-9e89-16a3c79ed47f.png"> | <img width="720" alt="image" src="https://user-images.githubusercontent.com/390760/170974674-99a51888-ba00-43fe-9b35-e35e1df5c63b.png">
